mail-manager/src/main/scala/it/pagopa/interop/commons/mail/InteropMailer.scala

```diff
@@ -0,0 +1,42 @@
+package it.pagopa.interop.commons.mail
+
+import courier.Defaults._
+import courier._
+import javax.mail.internet.InternetAddress
+import scala.concurrent.Future
+
+class InteropMailer private (sender: InternetAddress, mailer: Mailer) {
+
+  def send(mailData: Mail): Future[Unit] = sendMail(
+    mailData.recipients,
+    mailData.subject,
+    mailData.attachments match {
+      case Nil => Text(mailData.body)
```

Comment:
There will be cases where we want to send a body in HTML without any attachments.
Could be useful for this purpose to add a kind to the Mail class to identify if is a text or not?
What do you think? @TonioGela @galales

Reply:
That might be a good idea indeed

Reply:
makes sense